Website/src/rendering/src/assets/css/layout/sections.css

```diff
@@ -10,15 +10,33 @@
   }
 
   &--dark {
-    @apply bg-black;
+    background: url(../../img/background-pattern-dark.svg);
```

Comment:
By adding a background image here, you are adding it to all dark sections across the whole website. This includes:
- The Home page News and Media section
- Payment page Credit or Debit Card section

I guess that is not intended.

This is why section background images were usually defined in specific `/layout/section-something.css` files. This should be done here as well.